### ISSUE TITLE
Operational messages dumped as fatal errors

### ISSUE THREAD

**Issue by RavanH** [hiboutik/woocommerce-hiboutik#1]
I wonder if this is intentional but when there are products synced that have missing bar code, this type of error is thrown (plus stack trace) in the logs:
`
PHP Fatal error:  Uncaught Exception: Le produit  n'a pas du code des barres sur WooCommerce; synchronisation abandonee in /home/xxxx/public_html/wp-content/plugins/woocommerce-hiboutik-master/woocommerce-hiboutik.php:29
`

These can clutter the error log up pretty bad, considering these are not real errors, just plugin operation messages.

Also these seem to appear quite a lot:
`
PHP Fatal error:  Uncaught Exception: Pas des produits dans le POST d'url de callback; synchronisation abandonee in /home/xxxx/public_html/wp-content/plugins/woocommerce-hiboutik-master/woocommerce-hiboutik.php:24
`
Not sure what is causing those, but it looks like something that belongs in a plugin operation log (debug option maybe?) not in the main PHP error log.

**Comment by RavanH** [comment]
Also, variables like `{$$item['product_id']}` return nothing. I suppose the double dollar sign makes the error message useless...

**Comment by hiboutik** [comment]
Hello,

Thank you for the interest shown in our project.
We took into consideration your observations and suggestions. The new commit we've just made, introduces the features you requested, along with new logging options in the options page.
Logging will be made in the plugin's directory and a logrotate function is in place to take care of the clutter.
Hope it takes care of your problem. Let us know should you encounter any more problems.

Best regards
